gene2phenotype_project/gene2phenotype_app/models.py

```diff
@@ -105,25 +105,30 @@ class Meta:
             models.Index(fields=['lgd', 'phenotype']),
         ]
 
-class LGDPhenotypeComment(models.Model):
+class LGDPhenotypeSummary(models.Model):
+    """
+        Summary of the phenotypes reported in a publication for a G2P record.
+        This is not directly linked to a phenotype.
+    """
     id = models.AutoField(primary_key=True)
-    lgd_phenotype = models.ForeignKey("LGDPhenotype", on_delete=models.PROTECT)
-    comment = models.TextField(null=False)
-    date_created = models.DateTimeField(null=False)
-    is_public = models.SmallIntegerField(null=False, default=True)
-    user = models.ForeignKey("User", on_delete=models.PROTECT)
+    lgd = models.ForeignKey("LocusGenotypeDisease", on_delete=models.PROTECT)
```

Comment:
This table saves a summary/comment of all the phenotypes reported in the paper. It can be more than one phenotype so it won't be linked directly to any phenotype.

Reply:
The LGD-phenotype link is saved in the table `LGDPhenotype`